app/assets/stylesheets/admin/views/_editions-index.scss

```diff
@@ -25,6 +25,14 @@
     margin-left: 1em;
   }
 
+  .has-broken-links {
+    margin-left: 1em;
```

Comment:
You could consider making a helper class, something like.

```css
.label-spacing-left {
  margin-left: 1em;
}
```

Reply:
Yes, that was considered, but Tom said that he liked the individual names of the classes

Reply:
I think it's a good idea to move the style into a `.label-spacing-left` class, but then keep the other class names there too.

Reply:
Fair, if we keep as is be good to link them up like

```css
.class,
.other-class {
}
```